### ISSUE TITLE
In =CONCAT(), enter string with double quotes not working, its stop execution

### ISSUE THREAD

**Issue by Sonal1818** [myliang/x-spreadsheet#89]
In spreadsheet on =CONCAT() formula execution if i am typing SHIFT+" "  (shift key with double quote) then it will break execution and browser will crash.

**Comment by myliang** [comment]
![image](https://user-images.githubusercontent.com/718858/55392930-cfa56f80-556e-11e9-9d9d-06acaba7c184.png)
no problem like this

**Comment by Sonal1818** [comment]
Yes, It's working Now, but i have faced this issue. May be  this issue  is exist in previous version of xspreadsheet.

Thanks for quick reply.

**Comment by myliang** [comment]
OK

**Comment by Sonal1818** [comment]
In =CONCAT() function, position of output string is changed. Ex: If I am typing =CONCAT("Hello","Myliang") then it's display "MyliangHello" but correct output is "HelloMyliang".
